packages/template-retail-react-app/app/components/experience/region/index.jsx

```diff
@@ -21,7 +21,7 @@ export const Region = (props) => {
     const {id, components} = region
 
     return (
-        <div id={id} className={`region ${className}`} {...rest}>
+        <div id={id} className={`region ${className}`} style={{marginBottom: 15}} {...rest}>
```

Comment:
That grid spacing is jsut something that I added because it was easy to do with ChakraUI. I don't want to be opinionated to much with styling and I was hoping to loop back on a clean up ticket to address any styling questions. So I've removed the space between forthe moment.